### PR TITLE
Changing shield examples

### PR DESCRIPTION
**Pull request by gijsio**

### CODE REVIEW

File: content/datasheets/expansionboards/pyscan.md [hunk 1]
```diff
@@ -8,15 +8,31 @@ aliases:
...
-![](/gitbook/assets/pyscan-new.png)
+The Pyscan shield allows you to scan RFID and NFC tags and includes an accelerometer and light sensor.
 
+## Getting started
+1. Find the libraries for the Pyscan in the [Pycom libraries](https://github.com/pycom/pycom-libraries/shields/) repository on Github. 
+1. Download the files and extract them into the project folder in Pymakr
+1. Click the `upload project to device` button. This will store all necessary files on the device and allow you to import them in the example `main.py`.
+1. Check the REPL:
+```
+example output
```
Comment (peter-pycom) [inline content/datasheets/expansionboards/pyscan.md]: I assume this is a draft right?! ... I'm marking it, let me know otherwise

**Comment by gijsio** [comment]
TODO:
* Add the correct examples in pycom-libraries
* Have ready-downloadable releases for the shields
* Add more examples for the shields
* Test the examples
* Add figures in tracking example @matthewfelgate2 
*

**Review by peter-pycom — DISMISSED** [review]

**Comment by peter-pycom** [comment]
oh .... now I see that a) this is still draft and b) it has conflicts ...

**Review by peter-pycom — APPROVED** [review]

**Comment by peter-pycom** [comment]
more conflicts

**Pull request merged**